front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -172,7 +185,8 @@ const SimulationResultsV2 = ({ collapsedTimetable, spaceTimeData }: SimulationRe
             <div className="chart-container">
               <SpaceTimeChartWithManchette
                 operationalPoints={projectedOperationalPoints}
-                projectPathTrainResult={spaceTimeData}
+                projectPathTrainResult={formattedSpaceTimeDataToProjectPathTrainResult}
+                selectedProjection={trainUsedForProjectionSpaceTimeData?.id}
```

Comment:
Ah oups we have a naming issue. We should highlight the selectedTrain, and not the train used for the projection. We should change the name of the field in the manchette too
```suggestion
                selectedProjection={selectedTrainSchedule?.id}
```

Reply:
Done